game/input_additions.rpy

```diff
@@ -0,0 +1,468 @@
+# Original Paste code and Sub-Selections v2 by LegendKiller21
+# Copy, Cut, Select All, and Original Markers for Sub-Selections and Sub-Selections v2 optimizations by LordBaaa
+#
+# Although this is heavily edited, original by LegendKiller21 and LordBaaa as mentioned above
+# original - https://github.com/Legendkiller21/MAS-Submods-Paste
+
+
+init 1 python:
+    config.keymap['input_paste'] = ['ctrl_K_v']
+    config.keymap['input_copy'] = ['ctrl_K_c']
+    config.keymap['input_cut'] = ['ctrl_K_x']
+    config.keymap['input_select_all'] = ['ctrl_K_a']
+    config.keymap['input_move_select_left'] = ['ctrl_K_LEFT', 'ctrl_repeat_K_LEFT']
+    config.keymap['input_move_select_right'] = ['ctrl_K_RIGHT', 'ctrl_repeat_K_RIGHT']
+    config.keymap['input_move_select_home'] = ['ctrl_K_HOME']
+    config.keymap['input_move_select_end'] = ['ctrl_K_END']
+
+init 999 python in Input_overwrite:
```

Comment:
Small consistency thing. Also we should going forward prefix our vars/classes/funcions/etc. with a prefix to indicate it's global. (Kinda like a prod name before the package)
```suggestion
init 999 python in jn_input_overwrite:
```